agreements/service_level_agreement.md

```diff
@@ -0,0 +1,96 @@
+# Service Level Agreement
+
+## 10 April 2017
+
+1. ### Introduction
+
+    1. dxw has created this Service Level Agreement to define the minimum
+       standard of responsiveness that clients can expect from us when using
+       dxw's helpdesk.
+
+    2. The responsiveness targets given in this agreement are maximal. dxw will
+       make best endeavours to respond to all issues as quickly as possible,
+       notwithstanding the requirements of this agreement.
+
+
+2. ### Priority
+
+    1. There are four levels of priority that apply to tickets, as described
+       below.
+
+    2. A ticket's priority is set by dxw, based on our understanding of the
+       impact and importance of the problem.
+
+    3. Clients are requested to describe the problem and its impact as clearly
+       as possible, in order to allow us to set priority appropriately.
+
+    4. Tickets are set to Normal priority as a matter of course on receipt.
+
+    5. Tickets sent to support-emergency@dxw.com are set to Urgent priority on
+       receipt.
+
+    6. The receipt of an Urgent ticket will result in the immediate alerting of
```

Comment:
No, I don't think so - the subsequent sections are focussed on the definitions of the respective priorities and our target response times.